src/lightning/pytorch/trainer/connectors/data_connector.py

```diff
@@ -239,28 +239,17 @@ def _prepare_dataloader(
         """This function handles the following functionalities:
 
         - Injecting a `DistributedDataSamplerWrapper` into the `DataLoader` if on a distributed environment
-        - Wrapping the datasets and samplers into fault-tolerant components
         - Wrapping the dataloader based on strategy-specific logic
         """
         if isinstance(dataloader, CombinedLoader):
```

Comment:
A follow-up will address how we interact with the `CombinedLoader` across the codebase